test/documentation/manpages/test.bats

```diff
@@ -9,11 +9,21 @@ setup() {
 # }
 
 @test "Check manual pages are up to date" {
- for rst in ${BATS_TEST_DIRNAME}/../../../docs/*.[1-9].rst
- do
-   echo "$rst"
-   rst2man.py "$rst" | cmp  "${rst%.rst}" -
- done
+  for rst in ${BATS_TEST_DIRNAME}/../../../docs/*.[1-9].rst
+  do
+    echo "$rst"
+    rst2man.py "$rst" | cmp  "${rst%.rst}" -
+  done
 }
 
+@test "Check manual pages are clean" {
+  type -t git >/dev/null || skip
+  cd $(git rev-parse --show-toplevel)   
```

Comment:
Should we use this for the first test as well? Looks cleaner than `${BATS_TEST_DIRNAME}/../../../docs.....`